task/orka-full/0.1/README.md

```diff
@@ -0,0 +1,177 @@
+
+```sh
+./add-orka-creds.sh --delete
+./add-ssh-creds.sh --delete
+```
+
+To create a Kubernetes secret in a custom namespace, run the following commands against your preferred namespace:
+
+```sh
+NAMESPACE=tekton-orka EMAIL=<email> PASSWORD=<password> ./add-orka-creds.sh --apply
+NAMESPACE=tekton-orka SSH_USERNAME=<username> SSH_PASSWORD=<password> ./add-ssh-creds.sh --apply
+```
+
+To remove the secrets from the custom specify, run the following commands against the namespace:
+
+```sh
+NAMESPACE=tekton-orka ./add-orka-creds.sh --delete
+NAMESPACE=tekton-orka ./add-ssh-creds.sh --delete
+```
+
+**Manual setup**
+
+If you want to create the Kubernetes secrets manually, you can use the following example configuration. Make sure to provide the correct credentials for your Orka environment and the base image.
+
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orka-creds
+type: Opaque
+stringData:
+  email: tekton-svc@macstadium.com
+  password: p@ssw0rd
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orka-ssh-creds
+type: Opaque
+stringData:
+  username: admin
+  password: admin
+```
+
+**Using an SSH key**
+
+If using an SSH key to connect to the VM instead of an SSH username and password, complete the following:
+
+1. Copy the public key to the VM and commit the base image.
+2. Store the username and private key in a Kubernetes secret:
+
+```sh
+kubectl create secret generic orka-ssh-key --from-file=id_rsa=/path/to/id_rsa --from-literal=username=<username>
+```
+
+See also: [`use-ssh-key`](samples/use-ssh-key.yml) example
+
+### Workspaces
+
+* **orka**: An Orka environment against which to perform all operations. The environment parameters are configured with the `Task` parameters.
+
+### Parameters
+
+#### Common parameters
+
+| Parameter | Description | Default |
+| --- | --- | ---: |
+| `base-image` | The Orka base image to use for the VM config. | --- |
+| `cpu-count` | The number of CPU cores to dedicate for the VM. Must be 3, 4, 6, 8, 12, or 24. | 3 |
+| `vcpu-count` | The number of vCPUs for the VM. Must equal the number of CPUs, when CPU is less than or equal to 3. Otherwise, must equal half of or exactly the number of CPUs specified. | 3 |
+| `vnc-console` | Enables or disables VNC for the VM. | false |
+| `script` | The script to run inside of the VM. The script will be prepended with `#!/bin/sh` and `set -ex` if no shebang is present. You can set your shebang instead (e.g., to run a script with your preferred shell or a scripting language like Python or Ruby). | --- |
+| `copy-build` | Specifies whether to copy build artifacts from the Orka VM back to the workspace. Disable when there is no need to copy build artifacts (e.g., when running tests or linting code). | true |
+| `verbose` | Enables verbose logging for all connection activity to the VM. | false |
+| `ssh-key` | Specifies whether the SSH credentials secret contains an [SSH key](#using-an-ssh-key), as opposed to a password. | false |
```

Comment:
The link here won't work, since the Using an SSH Key section is not a header. Consider changing that section name to a minor header, e.g. `#### Using an SSH key` or removing the link here. Personally I think the first option is preferable as having the in-doc link is helpful.